osquery/distributed/distributed.cpp

```diff
@@ -173,15 +175,14 @@ Status Distributed::acceptWork(const std::string& work) {
 
     auto& queries = tree.get_child("queries");
     for (const auto& node : queries) {
-      DistributedQueryRequest request;
-      request.id = node.first;
-      request.query = queries.get<std::string>(node.first, "");
-      if (request.query.empty() || request.id.empty()) {
+      if (queries.get<std::string>(node.first, "").empty() ||
```

Comment:
Since you accessing and casting `node.first` here and other places, why not:
```cpp
auto query = queries.get<std::string>(node.first, "");
```

Then you'll prevent the second cast.